### PR TITLE
fix(wallet-mobile): dapp discover ui issues

### PR DESCRIPTION
**Pull request by lephuochoai**

### CODE REVIEW

File: apps/wallet-mobile/src/features/Discover/common/LabelCategoryDApp.tsx [hunk 1]
```diff
@@ -44,7 +41,7 @@ const useStyles = () => {
   })
 
   const colors = {
-    gradientColor: color.bg_gradient_1,
+    gradientBg: ['#17D1AA', '#1ACBAF', '#21B8BC', '#2E9BD3', '#3F71F1', '#475FFF'],
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Discover/common/LabelCategoryDApp.tsx]: @michaeljscript can you check with Dima if this gradient is not in Figma and why?

File: apps/wallet-mobile/src/features/Discover/useCases/BrowseDapp/BrowserTabBar.tsx [hunk 2]
```diff
@@ -111,7 +111,7 @@ const useStyles = () => {
       justifyContent: 'space-between',
       gap: 16,
       ...atoms.px_lg,
-      paddingVertical: 12,
+      paddingTop: 12,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Discover/useCases/BrowseDapp/BrowserTabBar.tsx]: ...atoms.pt_md ?

File: apps/wallet-mobile/src/kernel/i18n/locales/en-US.json [hunk 3]
```diff
@@ -1062,6 +1063,8 @@
   "discover.unverifiedDappModal.understand": "I understand",
   "discover.unverifiedDappModal.disclaimerModalText": "You are accessing a third-party Cardano application that is not verified by EMURGO, the developers of Yoroi Wallet, or any of its affiliated divisions. Make sure this DApp is reliable to stay safe from malicious apps in a sustainable way.",
   "discover.unverifiedDappModal.disclaimerModalTitle": "Disclaimer",
+  "discover.confirmDisconnectDApp.disconnectDApp": "Disconnect DApp",
+  "discover.confirmDisconnectDApp.confirmDisconnectDAppDescription": "Here's an alert description that uses \nAuto Layout!",
```
Comment (michalszorad) [inline apps/wallet-mobile/src/kernel/i18n/locales/en-US.json]: I'll confirm the text with design team

File: apps/wallet-mobile/src/features/Discover/common/LabelCategoryDApp.tsx (hunk 1 above)
Comment (michalszorad) [reply apps/wallet-mobile/src/features/Discover/common/LabelCategoryDApp.tsx]: Waiting for a reply https://www.figma.com/design/KeBVo6rWNYFLXwTphz9lHi?node-id=16084-127102#843199905

File: apps/wallet-mobile/src/kernel/i18n/locales/en-US.json (hunk 3 above)
Comment (michalszorad) [reply apps/wallet-mobile/src/kernel/i18n/locales/en-US.json]: Can we change the text to "Are you sure you want to disconnect this DApp?"

File: apps/wallet-mobile/src/features/Discover/common/LabelCategoryDApp.tsx (hunk 1 above)
Comment (michalszorad) [reply apps/wallet-mobile/src/features/Discover/common/LabelCategoryDApp.tsx]: Hello @lephuochoai can we update the color to be static blue (with no gradient) as is now updated in the designs please?

Comment (lephuochoai) [reply apps/wallet-mobile/src/features/Discover/common/LabelCategoryDApp.tsx]: Okie, I'm checking

File: apps/wallet-mobile/src/features/Discover/useCases/BrowseDapp/WebViewItem.tsx [hunk 4]
```diff
@@ -42,7 +41,7 @@ export const WebViewItem = ({tab, index, nativeHeight}: Props) => {
 
   const {initScript, handleEvent} = useConnectWalletToWebView(wallet, webViewRef)
 
-  const visibleAreaHeight = nativeHeight ?? SCREEN_HEIGHT - insets.top - insets.bottom
+  const visibleAreaHeight = SCREEN_HEIGHT - insets.top
```
Comment (michalszorad) [inline apps/wallet-mobile/src/features/Discover/useCases/BrowseDapp/WebViewItem.tsx]: Hello, the screen size seems a bit cut off on android

Screenshot 2024-06-18 at 11 12 45

Comment (lephuochoai) [reply apps/wallet-mobile/src/features/Discover/useCases/BrowseDapp/WebViewItem.tsx]: Screenshot 2024-06-18 at 17 22 25

On my Android simulator, it is ok

Comment (rahulnr7) [reply apps/wallet-mobile/src/features/Discover/useCases/BrowseDapp/WebViewItem.tsx:44]: Issue 4 & 9 are not fixed. Please review https://docs.google.com/document/d/1YcQOtbudJMe5S_YGdnRIc1smdqDmIo2k3TkvdKZ4xxs/edit

**Comment by rahulnr7** [comment]
Issue 4 & 9 are not fixed. Please review https://docs.google.com/document/d/1YcQOtbudJMe5S_YGdnRIc1smdqDmIo2k3TkvdKZ4xxs/edit

**Comment by lephuochoai** [comment]
> Issue 4 & 9 are not fixed. Please review https://docs.google.com/document/d/1YcQOtbudJMe5S_YGdnRIc1smdqDmIo2k3TkvdKZ4xxs/edit

I have already updated it. I set a minimum value of 40% of the height screen. Please check the last commit. Thanks

File: apps/wallet-mobile/src/features/Discover/useCases/BrowseDapp/WebViewItem.tsx (hunk 4 above)
Comment (michalszorad) [reply apps/wallet-mobile/src/features/Discover/useCases/BrowseDapp/WebViewItem.tsx]: I'm having a look, thanks

Comment (michalszorad) [reply apps/wallet-mobile/src/features/Discover/useCases/BrowseDapp/WebViewItem.tsx]: I'm getting the bottom cut off even if I change the native navigation. Could it be related to the camera? It seems like the only difference between the screenshots. I'm attaching all properties of the emulator

Screenshot 2024-06-19 at 11 09 40

```
Properties
avd.ini.displayname              Pixel_3a_API_34_extension_level_7_arm64-v8a
avd.ini.encoding                 UTF-8
AvdId                            Pixel_3a_API_34_extension_level_7_arm64-v8a
disk.dataPartition.size          6G
fastboot.chosenSnapshotFile
fastboot.forceChosenSnapshotBoot no
fastboot.forceColdBoot           no
fastboot.forceFastBoot           yes
hw.accelerometer                 yes
hw.arc                           false
hw.audioInput                    yes
hw.battery                       yes
hw.camera.back                   emulated
hw.camera.front                  emulated
hw.cpu.ncore                     1
hw.device.hash2                  MD5:0e6953ebf01bdc6b33a2f54746629c50
hw.device.manufacturer           Google
hw.device.name                   pixel_3a
hw.dPad                          no
hw.gps                           yes
hw.gpu.enabled                   yes
hw.gpu.mode                      auto
hw.initialOrientation            Portrait
hw.keyboard                      yes
hw.lcd.density                   440
hw.lcd.height                    2220
hw.lcd.width                     1080
hw.mainKeys                      no
hw.ramSize                       1536
hw.sdCard                        yes
hw.sensors.orientation           yes
hw.sensors.proximity             yes
hw.trackBall                     no
image.androidVersion.api         34
image.sysdir.1                   system-images/android-34/google_apis/arm64-v8a/
PlayStore.enabled                false
runtime.network.latency          none
runtime.network.speed            full
showDeviceFrame                  no
skin.dynamic                     yes
skin.path.backup                 /Users/michalszorad/Library/Android/sdk/skins/pixel_3a
tag.display                      Google APIs
tag.id                           google_apis
vm.heapSize                      256

```

Comment (lephuochoai) [reply apps/wallet-mobile/src/features/Discover/useCases/BrowseDapp/WebViewItem.tsx]: I just pushed some commits to fix some texts in dark mode, and the tab view that had been cut off under navigation. Please check it again. Thanks. cc @michaeljscript

**Comment by stackchain** [comment]
bringing in for other conflicts @rahulnr7 please continue on develop.

**Pull request merged**